src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/NativeRuntimeEventSource.Threading.NativeSinks.cs

```diff
@@ -228,11 +228,16 @@ private void ContentionStop(ContentionFlagsMap ContentionFlags, ushort ClrInstan
         [MethodImpl(MethodImplOptions.NoInlining)]
         public unsafe void ThreadPoolIOEnqueue(NativeOverlapped* nativeOverlapped)
         {
+#if TARGET_WINDOWS
```

Comment:
This should be inside the `if (IsEnabled(...` block

Reply:
Ah my bad, I'll update it